download.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Download
+---
+
+# Linux #
+
+**Gaffer {{ site.latest-release }}** (stable) compiled, containing [Appleseed {{ site.appleseed-version }}](https://appleseedhq.org). Compatible with Arnold {{ site.arnold-version }}, 3Delight {{ site.delight-version }}, and Tractor {{ site.tractor-version}}.
+
+<a href="https://github.com/gafferhq/gaffer/releases/download/{{ site.latest-release }}/gaffer-{{ site.latest-release }}-linux.tar.gz"><i class="fa fa-arrow-down"></i> Download (.tar.gz)</a>
```

Comment:
Can we make this (and the MacOS one) look like a button? It's probably the most important thing on the page, feels like it gets somewhat lost at the mo. Can we just use the styling we use on the links on the home page?